routing.rst

```diff
@@ -1336,15 +1336,19 @@ A possible solution is to change the parameter requirements to be more permissiv
 Route Aliasing
 --------------
 
-Route alias allow you to have multiple name for the same route:
+Route alias allows you to have multiple name for the same route
+and can be used to provide backward compatibility for routes that
+have been renamed.
+Let's say you have a route called ``product_show``
```

Comment:
Let's add a `:` at the end of this phrase. Also, we can move this phrase to the previous line to be part of the first paragraph.